Loader/Loader.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * File containing the Loader class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\Loader;
+
+/**
+ * JS and CSS loader.
+ */
+interface Loader
+{
+    /**
+     * Returns combined JS or CSS files.
+     *
+     * @param array $files File name array
+     *
+     * @return string The content of the combined files
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException
+     */
+    public function getCombinedFileContent(array $files);
```

Comment:
Naming: instead of "getCombined", we could go for the "combine" verb: ``combineFilesContent` (there are multiple files).